public/scripts/create-class.js

```diff
@@ -1,5 +1,19 @@
 'use strict';
 
+function handleDropDownClickClassCreate(index) {
+	$('.js-dropdown-btn-' + index).click(function(e) {
+		e.preventDefault();
+
+		$('.js-actions-dropdown-' + index).toggleClass('show');
+	});
+
+	$(window).click(function(e) {
```

Comment:
Desired functionality: user clicking anywhere in window other than on dropdown menu should be able to close that menu